app/views/users/sessions/redirect_from_magic_link.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h2 class="govuk-heading-l">Continue to your account</h2>
-
+    <h3 class="govuk-heading-l">Sign in successful</h3>
```

Comment:
Damn, we will need to sort out our headers I think. (as in h1, h2, h3, and where to use them to be semantically correct :P )
I would keep using `h2` here though.

Reply:
The wireframes show different sizes so I just picked the sizes that seemed to match the designs better. But yes, we may need to revisit this for consistency and to be semantically correct. I'll probably also get some feedback from the designer.